### DISCUSSION TITLE
[TESelect] size of options

### DISCUSSION THREAD

**Discussion by SergejGleithmann** [mdbootstrap/TW-Elements-React#173] [Support from community] [answered]
Hello, I have tried to use the `size` property of `TESelect` and set it to `sm`. While this looks good when the options are hidden I feel it would be more consistent if the font size and the padding of the options would shrink too. I was able to reduce the size via `theme/selectOptions`, however I cannot adjust the height because the hight apperas to be set via in-line Style and not vie TW classes.

I think it would be good if `size` would effect the options too or if I could set the height manually. Or is there another way I am not seeing?

**Comment by iprzybysz** [comment, marked as answer]
Hi @SergejGleithmann, you can set the height of options using  `optionHeight` property. Check out [Select API documentation.](https://tw-elements.com/docs/react/forms/select/#docsTabsAPI).

**Reply by SergejGleithmann** [reply]
Ah, I see. I somehow thaught this would be within `theme`. Thank you!
